llvm/lib/MC/MCGOFFStreamer.cpp

```diff
@@ -26,6 +27,23 @@ GOFFObjectWriter &MCGOFFStreamer::getWriter() {
   return static_cast<GOFFObjectWriter &>(getAssembler().getWriter());
 }
 
+namespace {
+// Make sure that all section are registered in the correct order.
+void registerSectionHierarchy(MCAssembler &Asm, MCSectionGOFF *Section) {
```

Comment:
LLVM style prefers `static` to anonymous namespace for internal functions.

Reply:
Remove the anonymous namespace, and added `static`.